content/applications/sales/sales/amazon_connector/setup.rst

```diff
@@ -1,104 +1,151 @@
-   either the product or the SKU of an offer. The offer can be manually created if it was not
-   automatically done yet. This is useful if you do not use the internal reference as the SKU or if
-   you sell the product under different conditions.
+   It's possible to force the pairing of a marketplace item with a specific product, by changing
+   either the product or the SKU of an offer to ensure they match. The offer can be manually created
+   if it was not automatically done yet.
+
+   This is useful if the internal reference is not used as the SKU, or if the product sells under
+   different conditions.
 
-If no database product with a matching internal reference is found for a given SKU or gift wrapping
-code, a default database product **Amazon Sale** is used. The same is done with the default product
-**Amazon Shipping** and the shipping code.
+If no database product with a matching internal reference is found for a given Amazon SKU or gift
+wrapping code, then a default database product, *Amazon Sale*, is used. The same is done with the
+default product *Amazon Shipping* if no database product is found for a given Amazon shipping code.
 
 .. note::
-   To modify the default products, activate the :ref:`developer mode <developer-mode>` and navigate
-   to :menuselection:`Sales --> Configuration --> Settings --> Connectors --> Amazon Sync -->
+   To modify default products, activate the :ref:`developer mode <developer-mode>`, and navigate to
+   :menuselection:`Sales app --> Configuration --> Settings --> Connectors --> Amazon Sync -->
    Default Products`.
 
-Configure taxes of products
-===========================
+Product tax configuration
+=========================
 
 To allow for tax reporting of Amazon sales with Odoo, the taxes applied to the sales order items are
-those set on the product or determined by the fiscal position. Make sure to have set the correct
-taxes on your products in Odoo or to have it done by a fiscal position, to avoid discrepancies in
-the subtotals between Seller Central and Odoo.
+those set on the product, or determined by the fiscal position.
+
+Make sure to have the correct taxes set on your products in Odoo, or have it done by a fiscal
+position, to avoid discrepancies in the subtotals between *Amazon Seller Central* and Odoo.
 
 .. note::
    As Amazon does not necessarily apply the same taxes as those configured in Odoo, it may happen
-   that order totals differ by a few cents from that on Seller Central. Those differences can be
-   resolved with a write-off when reconciling the payments in Odoo.
+   that order totals differ by a few cents between Odoo and *Amazon Seller Central*. Those
+   differences can be resolved with a write-off when reconciling the payments in Odoo.
 
 .. _amazon/add-new-marketplace:
 
 Add a new marketplace
 =====================
 
 :ref:`All marketplaces are supported by the Amazon Connector <amazon/supported-marketplaces>`, but
```

Comment:
Unnecessary ref.
It just points to the same sentence on another page so we had the reader click away from the page to read the same content twice 😁 

Better to just do this imo
```suggestion
All marketplaces are supported by the Amazon Connector. To add a new marketplace, proceed as
follows:
```